File: backend/internal/models/organization.go
package models

type Organization struct {
	ID            string
	Name          string
	Address       *string
	City          *string
	State         *string
	ZipCode       *string
	VATNumber     *string
	ContactEmail  *string
	ContactPhone  *string
	PecEmail      *string
	SdiCode       *string
	PersonnelInfo *string
}

File: src/frontend/src/models/Organization.ts
export interface OrganizationRequest {
  name: string;
  address: string;
  vat_number: string;
  state: string;
  city: string;
  zip_code: string;
  contact_email: string;
  pec_email: string;
  sdi_code: string;
  contact_phone: string;
  personnel_info: string;
  user_id: string;
}

export interface OrganizationResponse {
  message: string;
  organization_id: string;
}
